src/ru/skypro/Main.java

```diff
@@ -0,0 +1,19 @@
+package ru.skypro;
+
+import ru.skypro.task1.Book;
+import ru.skypro.task2.Author;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Book firstBook = new Book("Сказки", " Александр Пушкин", 2010);
+        Book secondBook = new Book("Стихи", " Михаил Леромонтов", 1995);
+        Author first = new Author("Александр", "Пушкин");
+        Author second = new Author("Михаил", "Лермонтов");
```

Comment:
сначала нужно создать объект автора, а затем его передавать в конструктор класса книги. Таким образом мы можем использовать одного автора на несколько книг
```suggestion
        Author first = new Author("Александр", "Пушкин");
        Author second = new Author("Михаил", "Лермонтов");
        Book firstBook = new Book("Сказки", first, 2010);
        Book secondBook = new Book("Стихи", second, 1995);

```